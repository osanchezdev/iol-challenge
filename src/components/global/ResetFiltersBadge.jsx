import React from 'react';
import {Link} from 'react-router-dom';
import {Badge} from 'react-bootstrap';

const ResetFiltersBadge = () => {
  return (
    <Link
      to={{
        pathname: '/',
        search: '',
      }}>
      <Badge pill variant="light">
        Reset filters
      </Badge>
    </Link>
  );
};

export default ResetFiltersBadge;
