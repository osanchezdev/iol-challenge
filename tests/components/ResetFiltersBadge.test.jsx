import * as React from 'react';
import {shallow} from 'enzyme';
import ResetFiltersBadge from '../../src/components/global/ResetFiltersBadge';

describe('Tests in <ResetFiltersBadge />', () => {
  test('<ResetFiltersBadge /> must be rendered correctly', () => {
    const wrapper = shallow(<ResetFiltersBadge />);

    expect(wrapper).toMatchSnapshot();
  });
});
