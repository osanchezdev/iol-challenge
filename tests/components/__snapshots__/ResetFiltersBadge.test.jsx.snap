// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <ResetFiltersBadge /> <ResetFiltersBadge /> must be rendered correctly 1`] = `
<Link
  to={
    Object {
      "pathname": "/",
      "search": "",
    }
  }
>
  <Badge
    pill={true}
    variant="light"
  >
    Reset filters
  </Badge>
</Link>
`;
